scripts/run-integration-tests.travis.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
 set -ev
 docker build -t one-app:at-test .
-if [ "${TRAVIS_PULL_REQUEST}" = "false" ] && [ "${TRAVIS_BRANCH}" = "master" ]; then
+if [ "${TRAVIS_PULL_REQUEST}" = "true" ] && [ "${TRAVIS_BRANCH}" = "prerelease" ]; then
```

Comment:
This would allow us to run the integration tests on the prerelease branch and once any pull request is merged to master, maintaining the existing functionality

Reply:
@nellyk I think we discussed about leaving the integration tests run in heroku for both the master branch (when a PR is merged) and for the prerelease branch so we have a constant healthy master as well as a final check when we are about to do a release

Reply:
yes the script maintains the existing functionality and add runs the tests in master

Reply:
Did you test it? the way that if statement looks it will only run against heroku for the pull request release not master

Reply:
ooh i just saw what you meant, forgot to update this. Thanks for catching this, my bad

Reply:
@nellyk Looks good on the prerelease branch, well done... did you test the other path when the branch is master and also when we only want to test in docker (the else path)?

Reply:
https://travis-ci.org/github/nellyk/one-app/jobs/672172748 - This would for for any other branch 
https://travis-ci.org/github/nellyk/one-app/jobs/671695418 this would for master

Reply:
> https://travis-ci.org/github/nellyk/one-app/jobs/671695418 this would for master